src/main/java/org/eclipsefoundation/adopters/resource/AdoptersResource.java

```diff
@@ -0,0 +1,90 @@
+/*******************************************************************************
+ * Copyright (C) 2020 Eclipse Foundation
+ * 
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ * 
+ * SPDX-License-Identifier: EPL-2.0
+ ******************************************************************************/
+package org.eclipsefoundation.adopters.resource;
+
+import java.util.Collections;
+import java.util.List;
+import java.util.stream.Collectors;
+
+import javax.inject.Inject;
+import javax.ws.rs.Consumes;
+import javax.ws.rs.GET;
+import javax.ws.rs.Path;
+import javax.ws.rs.PathParam;
+import javax.ws.rs.Produces;
+import javax.ws.rs.QueryParam;
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
+
+import org.eclipsefoundation.adopters.model.AdoptedProject;
+import org.eclipsefoundation.adopters.model.Project;
+import org.eclipsefoundation.adopters.service.AdopterService;
+import org.eclipsefoundation.adopters.service.ProjectService;
+
+/**
+ * ECA validation endpoint for Git commits. Will use information from the bots,
```

Comment:
Be careful when you copy/paste code from other projects. This comment has nothing to do in here.